<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDataProbeInfoWidget</class>
 <widget class="qSlicerWidget" name="qSlicerDataProbeInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="LayerInfo">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="ViewerColor">
        <property name="text">
         <string>ViewerColor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ViewerName">
        <property name="text">
         <string>ViewerName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ViewerRAS">
        <property name="text">
         <string>ViewerRAS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ViewerOrient">
        <property name="text">
         <string>ViewerOrient</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ViewerSpacing">
        <property name="text">
         <string>ViewerSpacing</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="LayerGridContent">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="L_LayerId">
        <property name="text">
         <string>L</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="L_LayerName">
        <property name="text">
         <string>L_LayerName</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="L_LayerIJK">
        <property name="text">
         <string>L_LayerIJK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="L_LayerValue">
        <property name="text">
         <string>L_LayerValue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="F_LayerId">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="F_LayerName">
        <property name="text">
         <string>F_LayerName</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="F_LayerIJK">
        <property name="text">
         <string>F_LayerIJK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="F_LayerValue">
        <property name="text">
         <string>F_LayerValue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="B_LayerId">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="B_LayerName">
        <property name="text">
         <string>B_LayerName</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="B_LayerIJK">
        <property name="text">
         <string>B_LayerIJK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="B_LayerValue">
        <property name="text">
         <string>B_LayerValue</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
